examples_tests/49.ComputeFFT/fft_convolve_ifft.comp

```diff
@@ -13,23 +13,16 @@ layout(local_size_x=_NBL_GLSL_WORKGROUP_SIZE_, local_size_y=1, local_size_z=1) i
 
 // Input Descriptor
 
-layout(set=0, binding=0) readonly restrict buffer InputBuffer
+layout(set=0, binding=0) buffer InputOutputBuffer
```

Comment:
you can keep the `restrict` keyword, it means "no aliasing" that's all

Reply:
oh I didn't notice that